source/monitoring/monitoring-bandwidth-usage.rst

```diff
@@ -52,8 +52,8 @@ Built-in Graphs
 ---------------
 
 If overall per-interface usage is all that is required, there are
-built-in RRD graphs in pfSense software, which can be found under
```

Comment:
pfSense still uses RRD, it just uses the JSON output now and graphs using a javascript library.